src/TSMapEditor/Config/UI/Windows/PlaceWaypointWindow.ini

```
@@ -16,9 +18,19 @@ $X=EMPTY_SPACE_SIDES
 $Y=getBottom(lblDescription) + VERTICAL_SPACING
 $Width=getWidth(PlaceWaypointWindow) - (EMPTY_SPACE_SIDES * 2)
 
+[lblWaypointColor]
+$X=getX(tbWaypointNumber)
+$Y=getBottom(tbWaypointNumber) + VERTICAL_SPACING + 1
+Text=Color:
+
+[ddWaypointColor]
+$X=getRight(lblWaypointColor) + HORIZONTAL_SPACING
+$Y=getY(lblWaypointColor) - 1
+$Width=getRight(tbWaypointNumber) - getX(ddWaypointColor)
+
 [btnPlace]
 $Width=80
 $X=(getWidth(PlaceWaypointWindow) - getWidth(btnPlace)) / 2
-$Y=getBottom(tbWaypointNumber) + EMPTY_SPACE_TOP
+$Y=getBottom(lblWaypointColor) + EMPTY_SPACE_TOP
```

It's preferable to refer to non-label controls for positioning when possible, as labels are more affected by the editor's font.

```suggestion
$Y=getBottom(ddWaypointColor) + EMPTY_SPACE_TOP
```